parceler/src/main/java/org/parceler/internal/ParcelableAnalysis.java

```diff
@@ -725,9 +725,10 @@ private String getPropertyName(ASTMethod astMethod) {
         }
 
         for (String prepend : PREPENDS) {
-            if (methodName.startsWith(prepend)) {
+            if (methodName.startsWith(prepend) && methodName.length() - prepend.length() > 0) {
                 String name = methodName.substring(prepend.length());
-                return name.substring(0, 1).toLowerCase(Locale.getDefault()) + name.substring(1);
+                return name.substring(0, 1).toLowerCase(Locale.getDefault()) +
+                        (name.length() > 1 ? name.substring(1) : "");
```

Comment:
How about we disqualify methods like `get()`, `set()`, and `is()` from representting getters and setters in the `isGetter` and `isSetter` methods?

Reply:
Sure! I submitted a commit to change that, although I think the conditions got kind of messy in those methods. I'm unsure about making further changes because I could not get my IDE (Android Studio) to even compile the project. Please, do change anything in the PR you consider is improvable (and on that matter, compile the project 😬).

Reply:
I think you can revert these lines (731 and 732) since `"setA".substring(3).substring(1)` = `""`

Reply:
No worries.. this is a Maven project which Android Studio doesn't seem to support (Intellij IDEA does though!)

Reply:
Oh, right. I thought it would have thrown an `IndexOutOfBoundsException`. I just reverted it.